lambda-alb/src/AWS/Lambda/ALB.hs

```diff
@@ -56,17 +56,18 @@ instance JSON.FromJSON a => JSON.FromJSON (Request a) where
         . Text.encodeUtf8
 
 data Response = Response
-  { body       :: ResponseBody
-  , headers    :: Headers
-  , statusCode :: HTTP.Status
+  { body            :: ResponseBody
+  , isBase64Encoded :: Bool
```

Comment:
Per this [AWS article](https://aws.amazon.com/blogs/compute/handling-binary-data-using-amazon-api-gateway-http-apis/) media data aught to be base64 encoded and then indicated via `isBase64Encoded` option for AWS gateway to transform data back to original binary encoding.